articles/quickstart/native/android/index.yml

```diff
@@ -16,7 +16,7 @@ seo_alias: android
 default_article: dashboard-default
 articles:
   - getting-started
-  - 00-centralized-login
+  - 00-login
   - 01-embedded-login
```

Comment:
The rest of the articles refer to them as the first step they should do before that sample. I can remove them from the `index` in this PR or you can wait for me to update the rest of the articles in the other PR.